src/api/system/system.ts

```diff
@@ -731,6 +739,17 @@ export default class System extends EmitterBase<SystemEvents> {
         return data;
     }
 
+    /**
+     * Returns an array of all the installed runtime versions in an object.
+     * @return {Promise.<InstalledRuntimes>}
+     * @tutorial System.getInstalledRuntimes
+     */
+    // incompatible with standalone node process.
+    public getInstalledRuntimes() : Promise<InstalledRuntimes> {
+        return this.wire.sendAction('get-installed-runtimes')
+            .then(({ payload }) => payload.data);
+    }
+
     /**
```

Comment:
why return 'action' in response?

Reply:
good catch! I updated to just return `payload.data.runtimes` now.